### ISSUE TITLE
The model weights do not seem to be loaded correctly, and the classification accuracy of the pre-trained models in cheXpert varies greatly, with the average F1 of 0.42

### ISSUE THREAD

**Issue by XNLHZ** [RyanWangZf/MedCLIP#43]
Using the 5-category demo in the example, setting the logit threshold of 0.50, and using 500 frontal X-ray images from the cheXpert public test set for testing, the following test results are as follows:
![image](https://github.com/RyanWangZf/MedCLIP/assets/44829327/5103ece4-4b55-4c8a-bb57-8642b8a14079)
In addition, the classification results of the sample images do not match the displayed results:
![image](https://github.com/RyanWangZf/MedCLIP/assets/44829327/13f94282-6223-42e8-806c-b287d7778706)
The results should be: {'logits': tensor([[0.5154, 0.4119, 0.2831, 0.2441, 0.4588]]

**Comment by XNLHZ** [comment]
Has anyone had the same issue，thanks.
Very good work，I'd like to use it to do some follow-up work on the generation of radiology image reports.
